src/main/webapp/delf.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1" charset="utf-8">
+    <title>DELF</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <header>
+      <nav class="navbar navbar-inverse">
+        <div class="container-fluid">
+          <div class="navbar-header">
+            <a class="navbar-brand" href="index.html">ML Playground</a>
+          </div>
+          <ul class="nav navbar-nav">
+            <li class="active"><a href="index.html">Home</a>
+            <li><a href="delf.html">DELF</a></li>
+            <li><a href="imagenet.html">ImageNet Classification</a></li>
+            <li><a href="coco.html">COCO-SSD Object Detection</a></li>
+            <li><a href="visualization.html">t-SNE Visualization</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+
```

Comment:
Please remove the extra blank line (here and below).

Reply:
Ack, I've removed those lines